tests/providers/cncf/kubernetes/operators/test_kubernetes_pod.py

```diff
@@ -99,6 +99,33 @@ def test_image_pull_secrets_correctly_set(self, mock_client, monitor_mock, start
             [k8s.V1LocalObjectReference(name=fake_pull_secrets)],
         )
 
+    @mock.patch("airflow.kubernetes.pod_launcher.PodLauncher.start_pod")
+    @mock.patch("airflow.kubernetes.pod_launcher.PodLauncher.monitor_pod")
+    @mock.patch("airflow.kubernetes.kube_client.get_kube_client")
+    def test_image_pull_policy_correctly_set(self, mock_client, monitor_mock, start_mock):
```

Comment:
Shall we have one more test case, where we don't explicitly specify `image_pull_policy` then assert it against default value `IfNotPresent `?